src/Scoringline/SendinblueApi/Api/Email.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * This file is a part of scoringline sendinblue api package
+ *
+ * (c) Scoringline <joni@sendinblue.com>
+ *
+ * For the full license, take a look to the LICENSE file
+ * on the root directory of this project
+ */
+namespace Scoringline\SendinblueApi\Api;
+
+use Nekland\BaseApi\Api\AbstractApi;
+use Scoringline\SendinblueApi\Model\Email as EmailModel;
+
+class Email extends AbstractApi
+{
+    const API_URL = '/email';
+
+    /**
+     * @param EmailModel $emailModel
+     * @return array
+     * @throws \Exception
+     */
+    public function sendEmail(EmailModel $emailModel)
```

Comment:
why another need if we will not  use set method for attachment or images then automatically will not send  attachment. because default value is empty array.

Like 
If I used $email
    ->setTo(['to@example.com' => 'to name!'])
    ->setFrom(['to@example.com', 'to name!'])
    ->setSubject('Invitation')
    ->setText('You are invited for giving test');
    ->setHtml('This is the <h1>HTML</h1>')
;
then it will not send attachment and not give any error.